### ISSUE TITLE
W605 with LaTeX escape code in Matplotlib

### ISSUE THREAD

**Issue by jsdodge** [PyCQA/pycodestyle#975]
Pycodestyle flags an "invalid escape sequence '\g'" in the following code. Is there a way to correct this?

```python
import matplotlib.pyplot as plt

plt.text(0.5, 0.5, "Test $\gamma$")
plt.show()
```

**Comment by asottile** [comment]
please search the issue tracker, this will become a SyntaxError in future python versions:

```console
$ python3 -Werror -c '"Test $\gamma$"'
  File "<string>", line 1
SyntaxError: invalid escape sequence \g
```

if you need a tool which will autofix this for you (either by adding `r` where appropriate or `\` where appropriate), one such tool is [pyupgrade](https://github.com/asottile/pyupgrade#invalid-escape-sequences)

**Comment by jsdodge** [comment]
Thanks, @asottile. I did search the issue tracker and I did see the recommendation for the raw string, but my first attempt to correct it failed (it was a more complicated expression than my original MWE).

For anyone who finds this in a search, the relevant Matplotlib documentation is [here](https://matplotlib.org/3.3.3/tutorials/text/mathtext.html).

And for the more complicated use case that tripped me up, the correct format is below, which uses raw strings for LaTeX expressions and regular strings for newlines (the Matplotlib documentation indicates that Unicode can be included within a math text expression, as in `r'$\u23ce$'`, but this did not work for me).

```python
import matplotlib.pyplot as plt

plt.text(0.5, 0.5, r"Test $\gamma$" "\n" r"New line, $\gamma + 1$")
plt.show()
```
